src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java

```diff
@@ -121,6 +127,11 @@ public String getColumnValue(final VariantContext variant) {
 
     AS_RAW_MQRankSum { // TODO -- maybe rely on 1/1 for call_GT, also get rid of the | at the beginning
         public String getColumnValue(final VariantContext variant) {
+            // in the case where neither allele is reference, don't return a value
+            if (isGenotypeAllNonRef(variant.getGenotype(0))) {
```

Comment:
Really we want ref/something genotypes -- is there a check for hom-refs else where?  Because those are all non-ref....

Reply:
Hom-refs don't get here… the wouldn't be in the Vet table.